carlsim/test/compartments.cpp

```diff
@@ -174,6 +174,7 @@ TEST(COMPARTMENTS, spikeTimesCPUvsData) {
 *
 * This test makes sure that CPU mode and GPU mode of the compartment model produce the exact same spike times,
 * even with RK4 at 50 time steps.
+* FIX ME: TEST CURRENTLY FAILS AT numIntSteps = 30 (or 40 depending on machine) with small spike time deviation (2ms)
```

Comment:
Hey Ting, we've got a good start on NeuronMonitor, but it is unclear as to how long it will take to finish it. Meanwhile if you have time please try to debug the TEST(COMPARTMENTS, spikeTimesCPUvsGPU). You may know something about the code that I don't. That way no matter when Neuron Monitor is finished the Compartments will be ready to merge.